heat/core/operations.py

```diff
@@ -166,6 +168,51 @@ def allclose(x, y, rtol=1e-05, atol=1e-08, equal_nan=False):
     return torch.allclose(x._tensor__array, y._tensor__array, rtol, atol, equal_nan)
 
 
+def any(x, axis=None, out=None):
+    """
+    Returns a boolean indicating if any elements are non-zero.
```

Comment:
Docstring is generally good, but I would personally prefer it to be closer to the numpy one. See here:

https://docs.scipy.org/doc/numpy/reference/generated/numpy.any.html